tmd/view/view.py

```diff
@@ -327,8 +328,9 @@ def neuron(
     to_plot = []
 
     if rotation == "apical_dendrite":
-        angle = np.arctan2(nrn.apical_dendrite[0].get_pca()[0], nrn.apical_dendrite[0].get_pca()[1])
-        angle = np.arctan2(rotation[1], rotation[0])
+        rotation = np.arctan2(
+            nrn.apical_dendrite[0].get_pca()[0], nrn.apical_dendrite[0].get_pca()[1]
+        )
```

Comment:
These lines were very weird, I am not sure how the 'rotation' argument is supposed to be used.

Reply:
Maybe modify to "apical_alignment" and allow True / False